.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can you please exclude this file from the PR?

Reply:
Thought I removed it all.
Will do.

Reply:
Removed.